contrib/pg_tde/documentation/docs/release-notes/rc.md

```diff
@@ -0,0 +1,108 @@
+
+## Upgrade considerations
+
+`pg_tde` Release Candidate is not backward compatible with `pg_tde` Beta2 due to significant changes in code. This means you cannot directly upgrade from one version to another. You must [uninstall](../uninstall.md) `pg_tde` Beta2 first and then [install](../install.md) and configure the new Release Candidate version.
+
+## Changelog
+
+### New Features
+
+* [PG-1234](https://perconadev.atlassian.net/browse/PG-1234) - Added functions for separate global and database key management permissions 
+
+* [PG-1255](https://perconadev.atlassian.net/browse/PG-1255) - Added functionality to delete key providers
+
+* [PG-1256](https://perconadev.atlassian.net/browse/PG-1256) - Added single-tenant support via the default principal key functionality
+
+* [PG-1258](https://perconadev.atlassian.net/browse/PG-1258) - Added functions to display additional information  about principal keys / providers 
+
+* [PG-1294](https://perconadev.atlassian.net/browse/PG-1294) - Redesigned WAL encryption
+
+* [PG-1303](https://perconadev.atlassian.net/browse/PG-1303) - Deprecated tde_heap_basic access method
+
+## Improvements
+
+* [PG-858](https://perconadev.atlassian.net/browse/PG-858) - Refactored internal/principal key LWLocks to make local databases inherit a global key provider
+
+* [PG-1243](https://perconadev.atlassian.net/browse/PG-1243) - Investigated performance issues at a specific threshold and large databases and updated documentation about handling hint bits
+
+* [PG-1310](https://perconadev.atlassian.net/browse/PG-1310) - Added access method enforcement via the GUC variable
+
+* [PG-1361](https://perconadev.atlassian.net/browse/PG-1361) - Investigated and fixed pg_tde relocatability
+
+* [PG-1380](https://perconadev.atlassian.net/browse/PG-1380) - Added support for `pg_tde_is_encrypted()` function on indexes and sequences
+
+### Bugs Fixed
+
+
+* [PG-821](https://perconadev.atlassian.net/browse/PG-821) - Fixed the issue with `pg_basebackup` failing when configuring replication
+
+* [PG-847](https://perconadev.atlassian.net/browse/PG-847) - Fixed the issue with `pg_basebackup` and `pg_checksum` throwing an error on files created by `pg_tde` when the checksum is enabled on the database cluster
+
+* [PG-1004](https://perconadev.atlassian.net/browse/PG-1004) Fixed the issue with `pg_checksums` utility failing during checksum verification on `pg_tde` tables. Now `pg_checksum` skips encrypted relations by looking if the relation has a custom storage manager (SMGR) key.
+
+* [PG-1373](https://perconadev.atlassian.net/browse/PG-1373) Fixed the issue with potential unterminated strings by using the `memset()` or `palloc0()` instead of the `strncpy()` function.   
```

Comment:
Should be:

Fixed the issue with potential unterminated strings by using the `memcpy()` or `strlcpy()` instead of the `strncpy()` function.

Reply:
Updated